std_msgs/String imageDir 
---
std_msgs/Int32 nleft
std_msgs/Int32 nright
std_msgs/Int32 nInliers
std_msgs/String matchedDir
